// connect-auth/examples/example_keys_file.js
exports.bitbucketConsumerKey= "";
exports.bitbucketConsumerSecret= "";
exports.bitbucketCallbackAddress= "http://yourtesthost.com/auth/bitbucket_callback";
exports.fbId= "145603735541943";
exports.fbSecret= "bb4f5cf9b5ecc88e1880aee19d9eec69";
exports.fbCallbackAddress= "http://localhost:1337/auth/facebook_callback";
exports.getGlueId= "";
exports.getGlueSecret= "";
exports.getGlueCallbackAddress= "http://yourtesthost.com/auth/getglue_callback";
exports.ghId= "";
exports.ghSecret= "";
exports.ghCallbackAddress= "http://yourtesthost.com/auth/github_callback";
exports.twitterConsumerKey= "";
exports.twitterConsumerSecret= "";
exports.yahooConsumerKey= "";
exports.yahooConsumerSecret= "";
exports.yahooCallbackAddress= "http://yourtesthost.com/auth/yahoo_callback";
exports.googleConsumerKey= "";
exports.googleConsumerSecret= "";
exports.googleCallbackAddress= "http://yourtesthost.com/auth/google_callback";
exports.google2Id= "";
exports.google2Secret= "";
exports.google2CallbackAddress= "http://yourtesthost.com/oauth2callback";
exports.foursquareId= "";
exports.foursquareSecret= "";
exports.foursquareCallbackAddress= "http://yourtesthost.com/auth/foursquare_callback";
exports.janrainApiKey= "";
exports.janrainAppDomain= "yourrpxnowsubdomain";
exports.janrainCallbackUrl= "http://localhost/auth/janrain_callback";
exports.openIdCallback= "http://yourtesthost.com/verify";
